Doc/library/time.rst

```diff
@@ -120,13 +120,6 @@ An explanation of some terminology and conventions is in order.
 
 The module defines the following functions and data items:
 
-.. data:: altzone
```

Comment:
Please note that the time module also defines a few constants starting with `CLOCK_`.  Their descriptions should also go to the "Constants" section.

```
    CLOCK_MONOTONIC = 6
    CLOCK_MONOTONIC_RAW = 4
    CLOCK_PROCESS_CPUTIME_ID = 12
    CLOCK_REALTIME = 0
    CLOCK_THREAD_CPUTIME_ID = 16
```